etc/deploy/aws/aws-doco-script.sh

```diff
@@ -0,0 +1,279 @@
+#!/bin/bash
+
+# Preq: Run aws config to configure auth context profile before running this script.
+
+set -xeou pipefail
+
+jq --version || echo "This script required jq, install jq from https://stedolan.github.io/jq/download/"
+psql --version || echo "This script required psql, install psql from https://www.postgresql.org/download/"
+
+GLOBAL_TAG="jose-testing-4"
+
+CLUSTER_NAME="jose-pachyderm-cluster-4"
+AWS_REGION="us-east-2"
+AWS_PROFILE="default"
+
+BUCKET_NAME="jose-temp-bucket-4"
+S3_BUCKET_IAM_POLICY_NAME="jose-temp-bucket-iam-policy-4"
+S3_BUCKET_IAM_ROLE_NAME="jose-temp-bucket-iam-role-4"
+
+PODS_BUCKET_ACCESS_IAM_SA="jose-pods-bucket-access-iam-sa-4"
+
+EBS_CSI_DRIVER_POLICY_NAME="JoseAmazonEKS_EBS_CSI_Driver_Policy-4"
```

Comment:
Is this more-or-less a constant? If so, it would be worth creating it with a more generic name, I think.

Reply:
well i noticed that if you rerun the script with same policy name, aws will error and not let you create policy with same name.